backend/resolvers/types/entryType.js

```diff
@@ -0,0 +1,16 @@
+import { IMAGE_ENTRY, VIDEO_ENTRY, OTHER_ENTRY } from '../../constants'
+
+export default {
+  Entry: {
+    __resolveType (data, context, info) {
+      if (data.entryType === IMAGE_ENTRY) {
```

Comment:
I think this could be written more elegantly, but it wouldn't matter too much.

Reply:
agreed, but I dunno ... we can make it better if we think of what to do. For background, this is a magic method for GraphQL so it knows what type of the abstract `Entry` interface you're returning, so it extracts the right properties

Reply:
I don't think we need to change this now. It's pretty clear as is. If we need to add to it later, let's make sure it's clear.